modules/react/button/stories/visual-testing/stories_SecondaryButton.tsx

```diff
@@ -22,6 +22,7 @@ export const SecondaryButtonStates = (props: {theme?: PartialEmotionCanvasTheme}
           {value: 'inverse', label: 'Inverse'},
         ],
         size: [
```

Comment:
Can you [filter out](https://github.com/Workday/canvas-kit/blob/c11b43993e6e9a73555b6dcdbb74aa9025b489b1/modules/labs-react/common/lib/storybook-utils/permutateProps.ts#L5) some of the invalid combinations here as discussed?